sources/packages/backend/apps/workers/src/controllers/assessment/_tests_/e2e/assessment.controller-workflowWrapUp.e2e-spec.ts

```diff
@@ -113,6 +113,74 @@ describe("AssessmentController(e2e)-workflowWrapUp", () => {
     expect(expectedAssessment.updatedAt).toBeInstanceOf(Date);
   });
 
+  it(
+    `Should update assessment status to ${StudentAssessmentStatus.Completed} and update the wrap-up data` +
+      ` when the student assessment is currently in status ${StudentAssessmentStatus.InProgress}` +
+      ` and the application offering intensity is full-time.`,
+    async () => {
+      // Arrange
+      // Wrap-up data.
+      const workflowData = {
+        studentData: {
+          dependantStatus: "independant",
+        },
+      } as WorkflowData;
+      const eligibleApplicationAppeals = ["someEligibleAppeal"];
+      const savedApplication = await saveFakeApplication(
+        db.dataSource,
+        undefined,
+        {
+          offeringIntensity: OfferingIntensity.fullTime,
+          currentAssessmentInitialValues: {
+            workflowData,
+            eligibleApplicationAppeals,
+          },
+        },
+      );
+
+      savedApplication.currentAssessment.studentAssessmentStatus =
+        StudentAssessmentStatus.InProgress;
+      await db.studentAssessment.save(savedApplication.currentAssessment);
+
+      // Act
+      const result = await assessmentController.workflowWrapUp(
+        createFakeWorkflowWrapUpPayload(savedApplication.currentAssessment.id, {
+          workflowData,
+        }),
```

Comment:
I am not sure if I am following this. Please let me know if I am missing something.
Should the `createFakeWorkflowWrapUpPayload` be changed to also receive the `eligibleApplicationAppeals` at the same time, the `workflowData` and `eligibleApplicationAppeals` should not be saved during application creation at line 135/136?